content/07.pathogenesis.md

```diff
@@ -46,22 +46,30 @@ One severe example includes reports of cytokine storm-like responses in patients
 
 #### Reproduction Number and Dynamics of Transmission
 
-Accurate estimates of the reproduction number of a virus are crucial to understanding the dynamics of infection and to predict the effects of different interventions.
-The basic reproduction number and the timescale of infection (measured by the infectious period and the exposed period) govern population-level epidemic dynamics, with R~0~ being one of most critical epidemiological parameters [@isbn:9780691116174].
-The basic reproduction number, R~0~, is the expected number of new (secondary) infections caused by one infected person, assuming no time dependence and a wholly susceptible population [@doi:10.1111/j.1467-9574.1996.tb01482.x].
-R~0~ is a unitless number which in mechanistic models is a combination of parameters principally related to the transmission rate (rate of infection transmitting interactions for an infectious individual) and the infectious period [@doi:10.1126/science.abb5659; @isbn:9780691116174].
+Disease spread dynamics can be estimated using R~0~, the basic reproduction number, and R~t~, the effective reproduction number. 
+Thus, accurate estimates are crucial to understanding the dynamics of infection and to predict the effects of different interventions.
+R~0~ and the timescale of infection (measured by the infectious period and the exposed period) govern population-level epidemic dynamics, with R~0~ being one of most critical epidemiological parameters [@isbn:9780691116174]. 
```

Comment:
```suggestion
R~0~ and the timescale of infection (measured by the infectious period and the exposed period) govern population-level epidemic dynamics, with R~0~ being one of the most important epidemiological parameters [@isbn:9780691116174]. 
```

Reply:
we need this correction for grammar & because R0 is important in multiple ways. (At the very beginning when I offered suggestions for this section I put the basic concepts in my own words without reference because they are basic to field/research I've done, but then it was decided that it would make more sense to use references even for basic things because this manuscript is multidisciplinary and to shorten discussion at the beginning I just said I'd put things very close to original text of references which is fine if it's basic math or biology that cannot be said in multiple ways exactly but not as great if there are multiple ways to say it; I tend to have compound sentences which grammatically means more than 1 clause & when it was suggested for another biology section that for stylistic choices I break up sentences into basically 1-2 clauses each a biology method ended up being cut to 1 clause instead of 2-3 clauses of original suggestion which made it look very close to the reference used (it was exact biology terms so it was okay but still I preferred a different style) and also when compound sentences get so broken up they sometimes get far from the reference they belong to -- fortunately I caught that upon rereading the manuscript (I noticed it was so biologically exact & 1 sentence that it sounded more like something you'd read in a Methods section) so I fixed it to a shorter clearer (more basic/common terms) description in my own words; after seeing that I went back to this section (and other sections) to double-check if this might have occurred early on as well and caught the 2nd clause of the above sentence as being similar to the reference without necessity for exactness.  I never experienced this before (neither in drafts or final versions) -- probably because of the multi-author-multi-edits process without going back to check references; thankfully I double(+)-check writing and code.

Reply:
@LucyMcGowan @rdvelazquez please incorporate edit above. it's straightforward correction. thanks!

Reply:
The correction was incorporated here: https://github.com/greenelab/covid19-review/pull/550/commits/d37872a5cee09c157a54e75786989c5dd066f9a5.